product_docs/docs/pgd/5/reference/catalogs-internal.mdx

```diff
@@ -92,6 +92,7 @@ Per-node-group routing configuration options.
 | route_reader_max_lag    | bigint  | Maximum read lag accepted                                       |
 | route_writer_wait_flush | boolean | Wait for flush                                                  |
 
```

Comment:
Why are their quote marks on global, data ,and subscriber-only? Should they be single quotes and then tagged with back ticks? Or are they entered with quote marks, in which case you should still use backticks. Please advise how the user enters these so I can tag them correctly (or not).